frontend/src/services/frontpageService.jsx

```diff
@@ -1,68 +1,23 @@
-const getPosts = async () => {
-  // const response = await fetch('/api/post').then(respose => respose.json());
-
-  // return response;
-
-  return {
-    posts: [
-      {
-        id: 123,
-        title: 'title1',
-        content: 'contentcontentcontent',
-        date_created: '2020-03-11T03:26:09.094Z',
-        upvotes_clap: 10,
-        upvotes_laugh: 2,
-        upvotes_sad: 24,
-      },
-      {
-        id: 213,
-        title: 'title2',
-        content: 'contentcontentcontent2',
-        date_created: '2020-03-10T03:26:09.094Z',
-        upvotes_clap: 2,
-        upvotes_laugh: 6,
-        upvotes_sad: 0,
-      },
-      {
-        id: 312,
-        title: 'title3',
-        content: 'contentcontentcontent3',
-        date_created: '2020-03-12T03:26:09.094Z',
-        upvotes_clap: 0,
-        upvotes_laugh: 0,
-        upvotes_sad: 15,
-      },
-      {
-        id: 333,
-        title: 'title3',
-        content: 'contentcontentcontent3',
-        date_created: '2020-03-9T03:26:09.094Z',
-        upvotes_clap: 23,
-        upvotes_laugh: 0,
-        upvotes_sad: 15,
-      },
-    ],
-    response: 'ok',
-  };
+export const getPosts = async () => {
+  const response = await fetch('/posts').then(respose => respose.json());
+  return response;
 };
 
 // eslint-disable-next-line camelcase
-export const handleVote = async ({ id, date_created, upvote_type, upvote }) => {
-  const params = {
-    id,
-    date_created,
-    upvote_type,
-    upvote,
-  };
+export const handleVote = async ({ id, upvote_type, upvote }) => {
+  // const params = {
+  //   id,
+  //   upvote_type,
+  //   upvote,
+  // };
```

Comment:
I recommend using feature switches for this purpose, rather than commenting it out, but for now, this works